platform/o.n.core/test/qa-functional/src/org/netbeans/core/validation/ValidateClassFilesTest.java

```diff
@@ -120,4 +121,24 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
             }
         });
     }
+
+    private Map<String, Integer> calculateFileNameToVersion() {
+        Map<String, Integer> result = new HashMap<>();
+        for (Module module : Main.getModuleSystem().getManager().getModules()) {
+            for (Dependency dep : module.getDependencies()) {
+                if (dep.getType() == Dependency.TYPE_JAVA) {
+                    String javaVersion = dep.getVersion();
+                    if (javaVersion.startsWith("1.")) {
+                        break;
```

Comment:
is `break` correct here? You don't want to check the other dependencies in the list? (did you mean `continue`?)

Reply:
I meant `break`.  Shouldn't be more than one Java dependency in there? Should probably break later too.

Reply:
roger, just wanted to be sure.